.changeset/silent-dragons-sell.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/netlify': patch
```

Comment:
this needs an `'astro': patch` as well since an astro change is required.

Reply:
```suggestion
'@astrojs/netlify': patch
'astro': patch
```